index.js

```diff
@@ -32,11 +32,11 @@ const enableQueryTracing = ENABLE_QUERY_TRACING === "true"
 
 let server, isShuttingDown, hasBooted
 
-// Always load the source on startup so any file level issues surface sooner.
-require("./src")
```

Comment:
Ah, so this was the culprit?

Reply:
Yes, I would love to dig into this deeper and see if there is a better way to identify similar issues. Module loading and monkey patching bugs are so time consuming to investigate.

Reply:
Gah! I don't know how you tracked this down but thanks.